src/movies.js

```diff
@@ -1,22 +1,84 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+    const directorsArray = moviesArray.map(movie => movie.director)
+    return directorsArray
```

Comment:
Esto esta bien! pero poco a poco os vamos a ir pidiendo que vayais hacia esto, repito esto esta bien!
```suggestion
    return moviesArray.map(movie => movie.director)
```